packages/launcher/test/unit/linux_spec.ts

```diff
@@ -42,6 +43,24 @@ describe('linux browser detection', () => {
     return linuxHelper.detect(goal).then(checkBrowser)
   })
 
+  // https://github.com/cypress-io/cypress/issues/6669
+  it('detects browser if the --version stdout is multiline', () => {
+    stdout.withArgs('multiline-foo', ['--version'])
+    .resolves('Running without a11y support!\nfoo-browser v9001.1.2.3')
+
+    const goal = _.defaults({ binary: 'multiline-foo' }, _.find(goalBrowsers, { name: 'foo-browser' }))
+    const checkBrowser = (browser) => {
+      expect(browser).to.deep.equal({
+        name: 'foo-browser',
+        path: 'multiline-foo',
+        version: '9001.1.2.3',
+      })
+    }
+
+    // @ts-ignore
+    return linuxHelper.detect(goal).then(checkBrowser)
```

Comment:
I think in 2020 we can write this test as `async`

```
const found = await linuxHelper.detect(goal)
expect(found).to.deep.equal(...)
```

Reply:
or at least use 
```
return expect(linuxHelper.detect(goal)).to.eventually.deep.equal({...})
```

Reply:
sure sure, this is mostly a copy of the test above it, which was written in the dark ages of JS (2017)